src/dtscalibration/calibrate_utils.py

```diff
@@ -2346,16 +2335,15 @@ def match_sections(ds, matching_sections):
         hxs = ds.x.sel(x=hslice).size
         txs = ds.x.sel(x=tslice).size
 
-        assert (
-            hxs == txs
-        ), "the two sections do not have matching " "number of items: " + str(
-            hslice
-        ) + "size: " + str(
-            hxs
-        ) + str(
-            tslice
-        ) + "size: " + str(
-            txs
+        assert hxs == txs, (
+            "the two sections do not have matching "
+            "number of items: "
+            + str(hslice)
```

Comment:
f-strings could be even nicer here.

```py
f"number of items: {hslice}"
```
does the same thing. Not important (functionally it's the same), just a tip for future refactoring/code writing.